### DISCUSSION TITLE
clean up of issues

### DISCUSSION THREAD

**Discussion by markusschloesser** [markusschloesser/MackieC4_P3#52] [General]
@BilldarBagdar  if you don't mind, I'm gonna clean up our messy "issues", ok?
I will either make a new issues or delete, or open a discussion

**Comment by BilldarBagdar** [comment]
no problem, go for it.

**Reply by markusschloesser** [reply]
fyi; I decided to "hide" those comments, instead of deleting them (after I deleted a couple of course 😂)
